Remove bottom margin of components dropdown menu item

noahshrader opened the pull request:
## Description
Currently, there's inconsistent spacing between menu items in the toolbar dropdown menus. This commit removes the unnecessary bottom margin of the menu items in the "more" dropdown to match the spacing of the others.

## How has this been tested?
Reviewed the changes on local in major browsers on MacOS and all were consistent.

## Screenshots
The target issue has screenshots of before and after: https://github.com/WordPress/gutenberg/issues/23907

### Before
<img width="350" alt="Screen Shot 2020-07-27 at 11 43 20 AM" src="https://user-images.githubusercontent.com/3665539/88562298-80eed500-cffe-11ea-94c9-4f96c8443c37.png">

### After
<img width="366" alt="Screen Shot 2020-07-27 at 11 41 07 AM" src="https://user-images.githubusercontent.com/3665539/88562321-89dfa680-cffe-11ea-8377-d5453bc027dd.png">

## Types of changes
CSS-only

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/ -->
- [x] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
- [x] My code has proper inline documentation. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->
- [ ] I've included developer documentation if appropriate. <!-- Handbook: https://developer.wordpress.org/block-editor/ -->
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. <!-- React Native mobile Gutenberg guidelines: https://github.com/WordPress/gutenberg/blob/master/docs/contributors/native-mobile.md -->

noahshrader committed:
Remove bottom margin of components dropdown menu item

Currently, there's inconsistent spacing between menu items in the toolbar dropdown menus. This commit removes the unnecessary bottom margin of the menu items in the "more" dropdown to match the spacing of the others.

noahshrader committed:
Remove bottom margin of components dropdown menu item

Currently, there's inconsistent spacing between menu items in the toolbar dropdown menus. This commit removes the unnecessary bottom margin of the menu items in the "more" dropdown to match the spacing of the others.

noahshrader committed:
Remove bottom margin of components dropdown menu item

Currently, there's inconsistent spacing between menu items in the toolbar dropdown menus. This commit removes the unnecessary bottom margin of the menu items in the "more" dropdown to match the spacing of the others.

MichaelArestad committed:
Updated styling to match the post editor (#23525)

Updated styling to match the post editor

ockham committed:
Block Patterns: Override patterns that were registered by Core (#24004)

Unconditionally register Gutenberg's block patterns, potentially overriding ones that were registered by Core.

This is to make sure that Gutenberg is using its own block patterns (rather than Core's), since they might have changes that haven't been synced to Core yet.

bartczyz committed:
Refactor Tag Cloud block to use React Hooks (#23426)

* refactor Tag Cloud block to use React Hooks

* update onChange for showTagCounts

antonis committed:
[Mobile] Create cover block with only colour (from a default palette) without an image (#23833)

* Create cover block with only color (from a theme palette)

adamziel committed:
[LinkControl] Extract reusable parts (#23869)

* Slice-and-dice LinkControl to make parts of it reusable

* Remove out of scope changes to search create button

* Remove out of scope changes

* Add missing files

* Simplify handleSelectSuggestion

* Update LinkControlSearchResults props description

* Update snapshots

* Adjust directEntry handler usage to be consistent with the master branch

* Add mock

* Restore the intended order of results in useSearchHandler

* Use state in Creating Entities test

* Fix two other createSuggestion tests

* Fix all unit tests

* Remove the dependency on saveEntityRecord

* Fix unit tests

* Rename LinkOverview to LinkPreview

* Remove console.log

adamziel committed:
Prevent content loss after refreshing an editor with unsaved auto-draft post (#23928)

* Add isPostNew flag to localAutosaveGet/localAutosaveSet/localAutosaveClear to make them work with brand new unsaved posts

* Clear the autosave storage once auto-draft is saved

* Move cache eviction to useAutosavePurge

ceyhun committed:
[RNMobile] Fix bundling errors with metro (#23785)

ajitbohra committed:
Remove duplicate selectors (#23466)

mkaz committed:
Docs: Update plugin FAQ with versions link (#24005)

* Add FAQ for plugin versions

* Wording updates per review

SergioEstevao committed:
Ensure that the autosave happens on empty posts in the native apps. (#23233)

On the native apps, the publish button is managed by native code on
the apps (non Gutenberg/JS code). In order to update the status of the
publish button correctly we need to autosave even when there is no
content, in order to send that information across the native code.

enriquesanchez committed:
Update 'Quote' pattern (#23881)

* Update layout

* Added new line at end of file

* Update URL of avatar graphic

mkaz committed:
Image: Add the ability for a plugin to disable Image Editor (#23966)

ntsekouras committed:
Add block variations' keywords search support (#24040)

* add variation's keywords search support

* add search tests

* add docs

diegohaz committed:
Components: Fix Toolbar arrow key navigation in RTL contexts (#24043)

ellatrix committed:
RichText: restore aria-multiline (#24041)

ellatrix committed:
Fix block wrapper selector (#23991)

* Fix block wrapper selector

* Fix linting issue

Co-authored-by: Riad Benguella <benguella@gmail.com>

youknowriad committed:
Prevent JavaScript error when using arrow navigation in URLInput (#24047)

mateuswetah committed:
Docs/update button component props (#24013)

Commit:
Editor: Fix block highlight rendering after block is moved (#23425)

* Editor: Fix block outline rendering after block is moved

* Fix rendering for tests. Reduce Timeout from 250 -> 200 to improve UX.

* Fix rAF call with toggling highlight

* Resolve the E2E test issue, which appears to be caused by a double-mounting Toolbar

* Use reducer

Co-authored-by: Ella van Durpe <ella@vandurpe.com>

youknowriad committed:
Support disabling core block patterns (#24042)

afercia committed:
Fix Copy block button focus loss and try to remove the visually hidden textarea (#24022)

youknowriad committed:
Fix Inserter on the widgets screen (#24045)

Commit:
Use CSS variable for 'ui.brand' and 'ui.borderFocus' colors (#23936)

diegohaz committed:
Components: Add aria-expanded back to Panel (#24046)

ellatrix committed:
Inserter: fix line to show again (#20792)

* Inserter: fix line to show again

* Fix client id

* Split component

* Reorganise

Commit:
Rename color value ui.brand to ui.theme. Hardcode color variable fallbacks. (#24059)

SergioEstevao committed:
Move from release notes to changelogs (#24019)

* Rename release notes to changelog.

* Update to follow standard format.

* Add changelog to react-native-aztec and react-native bridge packages.

* Add information categorization labels.

youknowriad committed:
Bump plugin version to 8.6.0-rc.1

kjellr committed:
Fix broken links in block-based themes tutorial. (#24057)

ZebulanStanphill committed:
ReusableBlockConvertButton: use React hooks. (#24015)

* ReusableBlockConvertButton: use React hooks.

* Fix/simplify _isVisible initialization.

* Add types JSDoc to component.

atachibana committed:
Updated invalid block prompt (#24033)

Addison-Stavlo committed:
Post excerpt block - add color, fontSize, lineHeight, and text alignment. (#23945)

* add text align

* add wrapper and line-height inherit

* php format

* get this working in site editor?

* set initial readMore, get rid of extra parent component

* styles to ensure link color is shown

* no elipsis

* dont initialize for now /shrug

* add custom warning if no post found in context

* more simple error fix

* actually we need this warning... otherwise user can edit what cannot be dirtied/saved

* get rid of important

Addison-Stavlo committed:
Update Post Author block to use __experimentalColor and __experimentalLineHeight (#23044)

* add the new support flag

* remove the old

* remove unnecessary attributes

* updated colors in php

* add link color support

* update background classname

* gutting the unnecessary

* add style rules to add the set link color

* remove important style

enriquesanchez committed:
Change 'Two images' pattern to wide width (#23856)

* Change gallery to wide width

* Update two-images.php

Change the image URLs to reflect the proposed new location from the Core Trac ticket.

Co-authored-by: Jon Desrosiers <desrosj@users.noreply.github.com>

youknowriad committed:
chore(release): publish

 - @wordpress/annotations@1.20.2
 - @wordpress/block-directory@1.13.2
 - @wordpress/block-editor@4.3.2
 - @wordpress/block-library@2.22.2
 - @wordpress/blocks@6.20.2
 - @wordpress/components@10.0.2
 - @wordpress/compose@3.19.2
 - @wordpress/core-data@2.20.2
 - @wordpress/data-controls@1.16.2
 - @wordpress/data@4.22.2
 - @wordpress/e2e-tests@1.20.2
 - @wordpress/edit-navigation@1.3.2
 - @wordpress/edit-post@3.21.2
 - @wordpress/edit-site@1.11.2
 - @wordpress/edit-widgets@0.20.2
 - @wordpress/editor@9.20.2
 - @wordpress/format-library@1.22.2
 - @wordpress/interface@0.7.2
 - @wordpress/keyboard-shortcuts@1.9.2
 - @wordpress/list-reusable-blocks@1.21.2
 - @wordpress/notices@2.8.2
 - @wordpress/nux@3.20.2
 - @wordpress/plugins@2.20.2
 - @wordpress/react-native-editor@1.31.2
 - @wordpress/rich-text@3.20.2
 - @wordpress/server-side-render@1.16.2
 - @wordpress/viewport@2.21.2

noahtallen committed:
Site Editor: Remove more initial PHP state (#23775)

Commit:
Add isDestructive style to Button (#22921)

Co-authored-by: Dominik Schilling <dominikschilling+git@gmail.com>
Co-authored-by: jasmussen <joen@automattic.com>

mchowning committed:
Update passing of initial props on android (#23938)

* Use GutenbergProps object for initialProps

* Simplify GutenbergProps object

* Update demo app with GutenbergProps changes

Addison-Stavlo committed:
update align to textAlign (#24077)

noahtallen committed:
useBlockSync: Fix race condition when onChange callback changes (#24012)

* Fix race condition when onChange callback changed

* Add tests to cover the scenario

ramiy committed:
i18n: use __() to make the string translatable (#24075)

ocean90 committed:
Block Breadcrumb: Fix arrow direction in RTL (#24074)

lukewalczak committed:
[RNMobile] Fix jumping toolbar (#23684)

StevenDufresne committed:
Block Directory: Set icon image width to 100% for consistency. (#24086)

mcsf committed:
release.md: Fix "npm run" examples by adding '--' (#24100)

alshakero committed:
Increase z-index for submenu flyout (#24062)

Fixes: https://github.com/Automattic/wp-calypso/issues/41737

javidalkaruzi committed:
Make the padding for the panel content the same for all sides (#24084)

antonis committed:
Revert "Reduce spacing between label and slider control (#23580)" (#24109)

This reverts commit 2a67de0ca030b828af77f6bcc3c3732d3eea2c0a.

ntsekouras committed:
add missing prop in typedef (#24110)

SergioEstevao committed:
Mobile app specific bug report. (#24112)

* Mobile app specific bug report.

Add an issue template specific for reporting mobile apps bugs.

* Clarify the app version meaning.

* Address review comments.

* More review changes.

adamziel committed:
[URLInput] Make the control rendering customizable (#24115)

* Add renderControl prop to URLInput to make the usage of other types of controls possible (and easy)

* Simplify notation

* Dedupe getDerivedStateFromProps

* Move the className inside the rendered control

* Update jest snapshot

* Restore className to the input

* Update snapshot

adamziel committed:
Wrap LinkControlSearchInput in forwardRef (#24119)

Addison-Stavlo committed:
rename selector from wp-block (#24104)

epiqueras committed:
Template Part: Improve title editing interactions. (#23661)

* Core Data: Use the edited entity record for the multi-entity saving panel.

* Template Part: Select all text on name panel focus.

youknowriad committed:
Bump plugin version to 8.6.0

ocean90 committed:
I18N: Fix missing plural forms for block related strings (#24071)

ockham committed:
Perf Tests: Set up branch only once for all test suites (#24123)

youknowriad committed:
Remove deprecated blockType.context support (#24155)

youknowriad committed:
Fix the gallery buttons focus style (#24157)

youknowriad committed:
Fix Button block colors in the editor (#24153)

Addison-Stavlo committed:
Post Comments block - add style supports for text and background settings. (#24080)

* update block.json

* add textAlign in editor

* update wront context check / warnings

* allow comment html (links) to render in editor

* link colors to work in editor

* link colors to register on front end

* remove check for postType

* update editor rendering of comments

enriquesanchez committed:
Update style and contents, also adds a description (#24134)

enriquesanchez committed:
Updates Two buttons pattern (#24136)

* Updates content and style, also adds a description

* Update test snapshot

SergioEstevao committed:
Merge release 1.33.0 of GB-mobile to master (#24156)

* Release script: Update react-native-editor version to 1.33.0

* Update release notes.

* Update version numbers for packages.

* Revert "Reduce spacing between label and slider control (#23580)" (#24109)

This reverts commit 2a67de0ca030b828af77f6bcc3c3732d3eea2c0a.

Co-authored-by: Antonis Lilis <antonis.lilis@gmail.com>

enriquesanchez committed:
Updates 'Three buttons' pattern (#24137)

* Updates content and style, also adds a description

* Increase viewportWidth for better preview

* Add missing newline character

enriquesanchez committed:
Updates the 'Quote' pattern (#24140)

* Updates content and style, also adds a description

* Fix double quotes error

ramiy committed:
i18n: Merge similar translation strings in rss block (#24159)

* i18n: Merge similar translation strings in rss block

Moved from https://core.trac.wordpress.org/ticket/50739

* Add translation comment

enriquesanchez committed:
Updates 'Two columns of text' pattern (#24135)

* Updates contents and style, also adds a description

* Fix double arrow aligments

* Remove duplicate lines

enriquesanchez committed:
Updates 'Two images' pattern (#24139)

* Updates content and style, also adds a description

* Remove unnecessary double quotes

enriquesanchez committed:
Updates content and style, also adds a description (#24138)

enriquesanchez committed:
Adds 'Heading and paragraph' pattern (#24143)

* Adds 'Heading and paragraph' pattern

* Update layout of chapter and heading. Also description to reflect the change.

* Fix code spacing issues

MichaelArestad committed:
Add Post Tags block description and icon (#23496)

* Add a description and icon to the Post Tags block

MichaelArestad committed:
Add a description to the Post Title block (#23494)

* Add a description to the Post Title block.
* Added postTitle icon

enriquesanchez committed:
Updates 'Large header and a paragraph' pattern (#24142)

* Updates content, style and title, also adds a description

* Update gradient starting color

* Update contents of pattern and changed filename, title and description accordingly

* Fix double quotes issue

mapk committed:
Added a description for the post author block. (#24171)

tellthemachines committed:
Check nav is editable before counting items (#24146)

cameronvoell committed:
Added missing documentation links for contributor blog (#24179)

Commit:
[Mobile] - Cover block - Media editing (#23280)

* Mobile - Cover block - Media editing

* Mobile - Cover block - Fix upload placeholder

* Mobile - Cover media editing - Image edit button

* Mobile - Image editing button - Check for url before rendering

* Mobile - Cover - Show selected border with the overlay layer

* Mobile - Cover - Style improvements, Media editing support for onRemove

* Mobile - Cover - Remove extra line in style

* Mobile - Media editing - Support for more picker options

* Mobile - Cover - Update clear media label

* Mobile - Media Edit - Check if optionSelected is defined

* Mobile - Cover - Don't show the full screen modal preview if there is not a url set

* Mobile - Update release notes

* Mobile - Update changelog

Commit:
Mobile - Paste block - update label (#24096)

antonis committed:
[Mobile] Disable Page Template picker if Modal Layout Picker is enabled (#24061)

* MLP Android GlueCode

Commit:
Mobile - Cover - Hide media editing button for videos (#24189)

antonis committed:
[Mobile] Create cover block with custom colour (without an image) (#23870)

* Create cover block with custom color using the Color Picker

enriquesanchez committed:
Set heading line-height to 1.2 (#24177)

enriquesanchez committed:
Updates the 'Large header' pattern (#24141)

ntsekouras committed:
fix missing focus in tags (#24187)

talldan committed:
Fix dragging multiple blocks dowards resulted in blocks inserted in wrong position (#24183)

naogify committed:
Docs: Fix sample code of Block Preview in block registration docs (#23899)

* Fix sample code of Block Preview in block registration docs

* fix some

* Update docs/designers-developers/developers/block-api/block-registration.md

inner blocks array should contain block object

Co-authored-by: Ajit Bohra <ajit@lubus.in>

* Add a trailing comma.

Co-authored-by: Ajit Bohra <ajit@lubus.in>

claudiulodro committed:
Fix notices when block hasn't declared a support. (#24131)

mapk committed:
Post Date block: Added Post Date icon and updated block with icon and desc (#24178)

* Added Post Date icon and updated block with icon and desc.

* Added the post-date.js file.

* Fix linter

* Minor update to svg.

* Formatting adjustment

Co-authored-by: Sören Wrede <soerenwrede@gmail.com>

Designer023 committed:
Docs: Fix typo in block-code copy (#24211)

youknowriad committed:
Avoid focusing the block selection button on each render (#24195)

youknowriad committed:
Avoid rendering the clipboard textarea inside the button (#24194)

igorradovanov committed:
Docs: fix storybook demo link (#24212)

The current one resolves to 404.

talldan committed:
Fix dropdown menu focus loss when using arrow keys with Safari and Voiceover (#24186)

youknowriad committed:
Avoid focus style from being cut on the categories panel (#24197)

talldan committed:
Fix save shortcut in code editor (#24151)

dd32 committed:
Block Directory: Use local assets with automatic asset detection (#24117)

ocean90 committed:
Popover: Fix arrow color to match content border color (#24208)

youknowriad committed:
Update the editor landmark regions (#24196)

ocean90 committed:
Docs: Update default values for Popover props 'noArrow' and 'position' (#24207)

Copons committed:
Post Comments Counts: Add style attributes (#24167)

Update the Post Comments Counts block:

- Add support for colors, font size, line height, and text alignment.
- Convert the to light block wrapper.
- Output a `div` instead of a `span`.

ntsekouras committed:
Fix gradient RGBA/HSLA inputs' width (#24214)

* fix gradient rgba/hsla inputs width

* fix label by disableAlpha value

* change to px values

* css changes

enejb committed:
[GB-Mobile]: Add accessibilty label to the RichText component (#23057)

* Add accessibilty Lable and Hint to the rich Text component

* remove empty line that was added

* Update the hint to be less imperative

Co-authored-by: etoledom <etoledom@icloud.com>

* Remove the hint from RichText and update the Hint on the View to be more imparitive

* Remove the accessibilityHint since we are not using this feature anywhere just yet

* Add post type support clarification.

Fix rebase

Co-authored-by: etoledom <etoledom@icloud.com>

chipsnyder committed:
[Mobile] Generate static data to be used to develop Modal Layout picker (#24027)

* Adds helper method and data for creating base layouts for Gutenberg Modal Layour Picker

* Move Gutenberg Layout file to match existing folder structure

* Expose methods in the iOS Gutenberg pod

youknowriad committed:
Show the default style variation if none provided (#24217)

mapk reviewed (approved):
Looks good! The margins are consistent across the dropdowns that I tested.

![margins](https://user-images.githubusercontent.com/617986/88556957-77557500-cfde-11ea-90be-c6c998301772.gif)

Copons committed:
Post Comments Form: Add style attributes (#24162)

Update the Post Comments Form block:

- Add support for colors, font size, line height, and text alignment.
- Convert the to light block wrapper.
- Render warnings if the comment status of the element in context is unknown, or if the comments are closed.

desaiuditd committed:
Use Post Type label for Document tab in Settings Header (#17311)

* Introduce a filter to overrider Document label in Document Settings Header.

* Move the translator comment for ARIA label.

* Replace string concatenation with sprintf.

* Introduce useFilters React hook.

* Prefer template literal over string concatenation.

* Use PHP filter instead of JS filter.

* Use Post Type label for Custom Post Types.

Keep using Document for Posts and Pages.

* Re-use post type labels for Document label.

* Clean up & remove extra code.

* Fix typo.

* Use post type label in Document settings tab in sidebar for all post type.

* Fix e2e tests.

* Fix e2e tests.

* Update docs.

* Remove complexities around e2e tests.

* Clean up.

* Clean up.

* Update packages/edit-post/src/components/sidebar/settings-header/index.js

Co-authored-by: Pascal Birchler <pascal.birchler@gmail.com>

* Update packages/edit-post/src/components/sidebar/settings-header/index.js

Co-authored-by: Pascal Birchler <pascal.birchler@gmail.com>

* Fix nits.

* Fix nits.

* Fix nits.

* Add shim to override post type labels for Reusable blocks.

* Adds trac link.

* Fix i18n.

Co-authored-by: Pascal Birchler <pascal.birchler@gmail.com>

SergioEstevao committed:
Make shadow link color match text link color. (#24192)

* Make shadow link color match text link color.

* Update changelog.

oxyc committed:
fix #22550 and allow enter to insert line breaks even if template is locked (#23330)

* fix #22550 and allow enter to insert line breaks in richtext

* add e2e test

* Restore isSelected condition

Co-authored-by: Ella van Durpe <ella@vandurpe.com>

Addison-Stavlo committed:
'align' support flag - add server side block support for block alignment. (#24122)

* add function for blockAlign

* minor comment tweak

* add wrapper for template part block on front end

* update tests

noahshrader committed:
Merge branch 'fix/drop-down-spacing' of https://github.com/noahshrader/gutenberg into fix/drop-down-spacing

noahshrader committed:
Merge branch 'fix/drop-down-spacing' of https://github.com/noahshrader/gutenberg into fix/drop-down-spacing

mapk reviewed (changes requested):
Actually, it looks like this needs to be rebased. It currently shows 271 files changed.
